guides/v3.4.0/controllers/index.md

```diff
@@ -1,8 +1,8 @@
 ### What is a Controller?
 
-A [Controller](https://api.emberjs.com/ember/3.4/classes/Controller) is routable object which receives a single property from the Route – `model` – which is the return value of the Route's [`model()`](https://api.emberjs.com/ember/3.3/classes/Route/methods?anchor=model) method.
+A [Controller](https://api.emberjs.com/ember/3.4/classes/Controller) is routable object which receives a single property from the Route – `model` – which is the return value of the Route's [`model()`](https://api.emberjs.com/ember/3.4/classes/Route/methods?anchor=model) method.
```

Comment:
When there's `anchor` query parameter (say, the value is `foo`), I've been ensuring that the URL ends with the pattern `/foo?anchor=foo`.

I'm not sure how the app for `api.emberjs.com` handles URLs because `?anchor=foo` and `/bar?anchor=foo` will also scroll the page to `foo`. You will find both patterns mixed with the pattern above.

```suggestion
A [Controller](https://api.emberjs.com/ember/3.4/classes/Controller) is routable object which receives a single property from the Route – `model` – which is the return value of the Route's [`model()`](https://api.emberjs.com/ember/3.4/classes/Route/methods/model?anchor=model) method.
```